demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1628,18 +1680,20 @@ export class DemoMeetingApp implements
   }
 
   async openAudioOutputFromSelection(): Promise<void> {
-    if (this.defaultBrowserBehaviour.supportsSetSinkId()) {
+    if (true || this.defaultBrowserBehaviour.supportsSetSinkId()) {
```

Comment:
Is this intentional?

Reply:
Nope! Good catch. This was me provoking the error.